src/encoded/tests/test_batch_download.py

```diff
@@ -135,6 +135,39 @@ def test_batch_download_lookup_column_value(lookup_column_value_item, lookup_col
+    ('/batch_download/type=Experiment&files.file_type=bed bed6+', 'http://localhost/metadata/type%3DExperiment%26files.file_type%3Dbed%20bed6%2B/metadata.tsv'),
+    ('/batch_download/type=Annotation&organism.scientific_name!=Homo sapiens&organism.scientific_name=Mus musculus', 'http://localhost/metadata/type%3DAnnotation%26organism.scientific_name%21%3DHomo%20sapiens%26organism.scientific_name%3DMus%20musculus/metadata.tsv\nhttp://localhost/files/ENCFF015OKV/@@download/ENCFF015OKV.bed.gz'),
+])
+def test_batch_download_files(testapp, workbook, test_url, expected):
+    response = testapp.get(test_url)
```

Comment:
Do you think the 'batch_download' part should be abstracted from the test_url?  I can't decide.

Reply:
Thinking about it, yes. I can make a dictionary of tuples to extract away test_url and expected; and place them in a separate file. It makes the test more readable and hide the huge text.